src/data/README.md

```diff
@@ -18,7 +18,235 @@ TODO - describe the properties used in DevDot
 
 This file is only used in DevDot. It contains the data and content needed for each product's landing page (`/boundary`, `/consul`, etc.).
 
-TODO - describe the properties used
+### Editing product landing content
+
+Our `<product slug>-landing.json` pages take a "block"-based authoring approach for the main content area. This approach allows authors to write content in JSON instead of JSX. As an overview, each file has three properties, all are required:
+
+```json5
+{
+  "heading": "string", // required
+  "subheading": "string", // required
+  "blocks": [ /*  array of objects, required */ ], 
+}
+```
+
+Each of these top-level properties is addressed in more detail below.
+
+#### `heading` and `subheading`
+
+Set the heading and subheading shown on the page.
+
+<details>
+<summary>Example</summary>
+
+Source: 
+
+
+```json5
+{
+  "heading": "Waypoint Documentation", 
+  "subheading": "Use Waypoint to deliver a PaaS-like experience for Kubernetes, ECS, and other platforms.", 
+  "blocks": [ /* ... */ ],
+}
+```
+
+Result:
+
+![](https://user-images.githubusercontent.com/4624598/158818382-e78ea677-85c1-41aa-92b4-ca8714f06f2d.png)
+
+</details>
+
+#### `blocks`
+
+Each item in the `blocks` array represents a component on the page. Each of these items must have a `type` property, which can be one of the types listed below.
+
+<!-- block: type heading -->
+
+<details>
+<summary>Block type: <code>"heading"</code></summary>
+
+Heading blocks render HTML heading elements. Each block accepts the following properties:
+
+| Property | Type | Details |
+| --- | --- | --- |
+| `type` | `"heading"` | Block type |
+| `heading` | `string` | Text for the heading |
+| `level` | `2 \| 3 \| 4 \| 5 \| 6` | Semantic heading level, for example `2` becomes `<h2>`. Note that there is already an `<h1>` rendered for the page, so only values  should be used. |
+| `size` | `100 \| 200 \| 300 \| 400 \| 500` | Visual size of the heading. `500` is the largest size and `100` is the smallest. Visual size should generally reflect the semantic level, with `h2 = 300`, `h3 = 200`, and `h4` and below at the `100` size.
```

Comment:
💭 Might be worth linking to the `Heading` component here?

Reply:
✅ Linked!